qualtran/bloqs/qft/qft_phase_gradient.py

```diff
@@ -54,7 +54,9 @@ class QFTPhaseGradient(GateWithRegisters):
 
     @cached_property
     def signature(self) -> 'Signature':
-        return Signature.build(q=self.bitsize, phase_grad=self.bitsize)
+        return Signature(
+            [Register("q", QUInt(self.bitsize)), Register("phase_grad", QUInt(self.bitsize))]
```

Comment:
As discussed offline,phase gradient registers should be unsigned fixed point numbers with 0 integer bits.

Reply:
I'm tidying this up now, q and x are still unsigned ints right?

Reply:
I think keeping `q` and `x` as unsigned ints is probably fine for now. We can revisit later if needed.